cardano-config/src/Cardano/Config/ShelleyGenesis.hs

```diff
@@ -0,0 +1,235 @@
+{-# LANGUAGE StandaloneDeriving         #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings          #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
That is synonymouns with `-Wno-warn-orphans` which is more commonly used and more obvious.

Reply:
It's actually `-fno-warn-orphans`. I disagree, I prefer `-Wno-orphans`, as it's clear from the `-W` that it is a flag related to warnings (like `-Wall`, `-Wredundant-fconstraints` etc), instead of just *some* flag (like `-fprint-explicit-foralls`). See https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/using-warnings.html#warnings-and-sanity-checking.

Reply:
Most of the other files in this repo that use variants of that pagrma use `-fno-warn-orphans`. I think this should match.